### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by abhigunj000**
@abhigunj111 please check and confirm if this is good

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,31 +34,20 @@
 
       .b-example-divider {
         width: 100%;
-        height: 3rem;
-        background-color: rgba(0, 0, 0, .1);
-        border: solid rgba(0, 0, 0, .15);
-        border-width: 1px 0;
-        box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
```
Comment (abhigunj111) [inline index.html:41]: why did you change this ? any reason please explain

File: index.html [hunk 2]
```diff
@@ -34,31 +34,20 @@
...
-      }
 
-      .b-example-vr {
-        flex-shrink: 0;
```
Comment (abhigunj111) [inline index.html:45]: again why did you remove this ?

File: index.html [hunk 3]
```diff
@@ -34,31 +34,20 @@
...
-        width: 1.5rem;
-        height: 100vh;
-      }
 
       .bi {
         vertical-align: -.125em;
         fill: currentColor;
       }
-
+sxsxsxsxsxsxsxsxs
```
Comment (abhigunj111) [inline index.html:43]: why this garbage ?

File: index.html [hunk 4]
```diff
@@ -34,31 +34,20 @@
...
       .nav-scroller {
         position: relative;
         z-index: 2;
         height: 2.75rem;
         overflow-y: visible;
       }
-
+sxsxsxsxsxs
```
Comment (abhigunj111) [inline index.html:50]: another garbage ?

File: index.html [hunk 5]
```diff
@@ -90,7 +79,7 @@
         z-index: 1500;
       }
     </style>
-
+sxsxsxsx
```
Comment (abhigunj111) [inline index.html:82]: garbae ?

**Review by abhigunj111 — CHANGES_REQUESTED** [review]
please check my comments and make required changes

File: index.html (hunk 1 above)
Comment (abhigunj000) [reply index.html:41]: this was not needed hence i removed

File: index.html (hunk 4 above)
Comment (abhigunj000) [reply index.html:50]: looks fine

**Comment by abhigunj000** [comment]
i hope you are fien now

File: index.html (hunk 3 above)
Comment (abhigunj000) [inline index.html:43]: why did you add ?
